Source/Core/DolphinQt/SkylanderPortal/SkylanderModifyDialog.cpp

```diff
@@ -0,0 +1,344 @@
+// Copyright 2023 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "SkylanderModifyDialog.h"
+
+#include <QBoxLayout>
+#include <QCheckBox>
+#include <QDateTimeEdit>
+#include <QDialog>
+#include <QDialogButtonBox>
+#include <QLabel>
+#include <QLineEdit>
+#include <QMessageBox>
+#include <QStringConverter>
+#include <QValidator>
+
+#include <Core/IOS/USB/Emulated/Skylanders/Skylander.h>
+#include <Core/System.h>
+
+#include <DolphinQt/QtUtils/SetWindowDecorations.h>
```

Comment:
Includes from Dolphin itself should use `#include "file.h"`, not `#include <file.h>`.